src/components/NightLog.vue

```diff
@@ -79,11 +88,20 @@ export default {
             const url = this.$store.state.dev.active_api + '/nightlog/' + this.site
             axios.get(url).then(response => {
                 console.log(response)
+
+                // Save the note we just loaded
                 this.note = response.data
-                if (this.note !== '') {
+                this.note_is_loaded = true
+
+                // Check if the user has already seen this note
+                let note_id = this.note.site + this.note.created_timestamp
+                let last_read_note = window.localStorage.getItem('ptr_nightlog_last_read')
```

Comment:
If this night log is intended to be used in the control room, does this get logged to the local storage of the user's browser or the shared control room browser? (If it's the latter, would adding the logged in user's ID to the note_id circumvent that?)